dtm0/recovery.c

```diff
@@ -1278,7 +1278,7 @@ static void heq_post(struct recovery_fom *rf, enum m0_ha_obj_state state)
 	if (!rf->rf_is_local)
 		eolq_post(rf->rf_m,
 			  &(struct eolq_item) {
-			  .ei_type = EIT_HA,
+			  .ei_type = EIT_EOL,
```

Comment:
Can you post the fix in a separate PR ?
I think after 1503 PR is landed, we might not use DTM_ENABLE flag.

Reply:
Removed from this PR.

But I think this is a bug, isn't it?

Reply:
I mean can you create a separate PR for the bug .

Reply:
Yes, I will.